adele-booking/src/main/resources/static/app.js

```diff
@@ -31,15 +32,43 @@ function disconnect() {
     console.log("Disconnected");
 }
 
-function showEvent(event) {
-    $("#conversation").append("<tr><td>" + event.bookingId + "</td><td>" + event.positions[0].eventId + "</td><td>" + event.positions[0].sectorId + "/" + event.positions[0].id + "</td></tr>");
+function bookTickets() {
+    $.ajax({
+      type: "POST",
+      url: "/rs/api/bookTickets",
+      data: "{\"eventId\":1,\"sectorId\":1,\"positions\":[" + $("#seats").val() + "]}",
+      success: function(data) {console.log("BookingId: " + JSON.stringify(data))},
+      dataType: "json",
+      contentType: "application/json;charset=UTF-8"
+    });
+}
+
+function showEvent(eventText) {
+    $("#conversation").append("<p>" + eventText + "</p>");
+}
+
+function updateTicketStatus(event) {
+    $.each(event.positions, function(i, obj) {
+        $("#p-" + obj.id + " > td:nth-child(2)").text("BOOKED");
```

Comment:
@kelemena this raises the question: how should I know from a `{"bookingId":"84fdac14-6465-4126-96f5-85a6d0e581e0","positions":[{"eventId":1,"sectorId":1,"id":4}]}` message on the websocket, whether it's a book or a cancel event? :)